lib/providers/auth/app_initialization_provider.dart

```diff
@@ -17,78 +17,29 @@ final appInitializationProvider =
   try {
     var retryCount = ref.read(retryCounterProvider);
     if (retryCount < 2) {
-      _checkInternetConnectivity(context, ref);
+      var connectivityStatus = ref.read(connectivityStatusProvider);
+      if (connectivityStatus == ConnectivityStatus.isDisonnected) {
+        showSnackBar(context, StringConstants.connectivityError);
+      }
     }
     await _authProvider.initializeUser();
     var res = _authProvider.userRes;
```

Comment:
Can we try to avoid abbreviations? Res could be resource, response etc. It's a good idea to keep in mind that the code needs to be immediately understandable to anyone who's never seen it before :)